qiskit_optimization/applications/ising/max_cut_old.py

```diff
@@ -0,0 +1,83 @@
+# This code is part of Qiskit.
```

Comment:
I assume this 'old' file will just be deleted? (Same for the docplex old one)

Now it may be educational to have some direct to ising mapping in the maxcut app as its easy to follow. Of course there are papers out there about solving these type of problems with this mechanism so maybe it can be just simply removed, as you can get an ising from the QP, though its more work to follow/understand that path.

Reply:
I was thinking to delete them, but we haven't discussed about existing files, so I just renamed and kept them. If there is some good place, we can move them to there (or we can make some tutorial based on existing files in community-tutorial? I don't know).